assets/gift-card.scss.liquid

```diff
@@ -368,6 +368,17 @@ $gutter: 24px;
   vertical-align: baseline;
 }
 
+#QrCode {
```

Comment:
I know it's a bit late in the game, but I was wondering about the fact this says `#QrCode` and not `#qr-code`.

In the [Shopify repo](https://github.com/Shopify/shopify/blob/master/app/assets/stylesheets/gift-card.scss) they're using `#qr-code`.  Also, in [the issue the mark up Kevin asked us to copy from](https://github.com/Shopify/shopify-themes/issues/839), the id used is #qr-code.